src/e2e/puppeteer-environment.ts

```diff
@@ -9,28 +8,23 @@ const PuppeteerEnvironment: Environment = {
   async setup(global, options) {
     builtinEnvironments['happy-dom'].setup(global, options)
 
+    const args = [
+      '--deterministic-fetch',
+      '--disable-dev-shm-usage',
+      '--disable-features=IsolateOrigins',
+      '--disable-setuid-sandbox',
+      '--disable-site-isolation-trials',
+      '--no-first-run',
+      '--no-sandbox',
+      '--no-zygote',
+    ]
+
     const browser = await puppeteer
-      .launch({
-        headless: true,
-        // Disable Chrome features that crash GitHub Actions with "Protocol error (Target.createTarget): Target closed."
-        // See: https://stackoverflow.com/a/66994528/480608
-        // List of Chromium switches: https://peter.sh/experiments/chromium-command-line-switches/
-        args: [
-          '--deterministic-fetch',
-          '--disable-dev-shm-usage',
-          '--disable-features=IsolateOrigins',
-          '--disable-setuid-sandbox',
-          '--disable-site-isolation-trials',
-          '--no-first-run',
-          '--no-sandbox',
-          '--no-zygote',
-        ],
-        // Increase timeout to avoid: "TimeoutError: Timed out after 30000 ms while trying to connect to the browser! Only Chrome at revision r869685 is guaranteed to work."
-        timeout: 60000,
```

Comment:
I'm assuming this timeout is no longer needed because of the -1 timeout added to browserless?

Reply:
I'm afraid it is, this is the timeout controlled by puppeteer. The -1 is to disable timeouts on the browserless side.